README.md

```diff
@@ -92,8 +92,12 @@ Access the website by going to [localhost:9999](http://localhost:9999).
 This project is licensed under the MIT License - see the LICENSE.md file for details
 
 # Contributing
+ Use tabs for css and js files.
 
-Use tabs for css and js files.
+# IRC
+#nyaapant<span>su@irc<span>.rizon.n</span>et</span>
+
+#nyaapantsu-dev<span>@ir<span>c.rizon.n</span>et</span> (Serious buisness only)
```

Comment:
Github is retarded and tries to interpret these as email addresses, and won't allow me to actually use them as the IRC addresss that they are. It's a ugly hack but it would be really annoying to click a link  and it does something other than what you expect it to.

Alternatively I could just write on irc.rizon instead of using the @symbol, but it doesn't look as aesthetic.